docs/framework/langgraph.mdx

```diff
@@ -15,7 +15,9 @@ In this example, we will use LangGraph Agent to star a repository on Github usin
 pip install composio-langgraph
 ```
 ```bash TypeScript
-npm install composio-core langchain langgraph
+npm i @langchain/langgraph
+npm i composio-core 
+npm i @langchain/openai
```

Comment:
Consider using consistent npm command format. Either use `npm install` or `npm i` throughout, but mixing both styles can be confusing for users. Also, consider adding version numbers for these packages to ensure compatibility.